src/actions/dropbox/dropbox.ts

```diff
@@ -128,6 +128,7 @@ export class DropboxAction extends Hub.OAuthAction {
     })
 
     const payload = JSON.parse(plaintext)
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0"
```

Comment:
wat

Reply:
(since this project uses `dotenv`, you can just put `NODE_TLS_REJECT_UNAUTHORIZED=0` in a `.env` file at the root)

Reply:
oh whoops - just test code I use that accidentally made it up. thanks for the catch